justfile

```diff
@@ -7,7 +7,6 @@ maker_log_file := "$PWD/data/maker/regtest.log"
 # public regtest constants
 public_regtest_coordinator := "03507b924dae6595cfb78492489978127c5f1e3877848564de2015cd6d41375802@35.189.57.114:9045"
 public_regtest_esplora := "http://35.189.57.114:3000"
-public_coordinator_host:= "35.189.57.114"
```

Comment:
...but it doesn't work without it. It might work for you, because you keep your `.env` file. for casual runner (like myself or @luckysori ), we rely on this. 
If you don't mind, let's keep it :-)

Reply:
a bit of explanation: `run-regtest` should always use our regtest instance, it doesn't make sense to be changeable via the `.env` file (that's why I added it, and named it as such....).

Reply:
> ...but it doesn't work without it on my machine 😭 (nor for @luckysori )
Did you test it from my branch? 🙂 

> 🤔 Perhaps It might work for you, because you keep your .env file?
It's working because I am fetching the host from the config an not from the environment variable. I confess its a bit hidden in the code, but we already provide the host information through the `public_regtest_coordinator` variable.

The important piece of code is here in the `environment.dart`

```
 String p2pEndpoint = const String.fromEnvironment('COORDINATOR_P2P_ENDPOINT');
    if (p2pEndpoint.contains("@")) {
      final split = p2pEndpoint.split("@");
      coordinatorPublicKey = split[0];
      if (split[1].contains(':')) {
        host = split[1].split(':')[0];
        lightningPort = int.parse(split[1].split(':')[1]);
      }
    }
```
Unless I am missing something obvious, it should work for you too 🙂 .

> If you don't mind too much (minor redundancy after all), perhaps we could keep it as it is?
I don't mind at all, this is not critical at all.